### DISCUSSION TITLE
c7n-mailer - Possible to set SES Endpoint URL?

### DISCUSSION THREAD

**Discussion by iamperson347** [cloud-custodian/cloud-custodian#7394] [General]
I've been looking through the documentation for c7n-mailer to see if it is possible to set the endpoint url for SES. I see mention of setting the endpoint url for SQS (useful for VPC Endpoints, etc.), but it would also be good to be able to specify the SES endpoint URL. Besides VPC Endpoint usage, another potential use case for this is to force boto to use a FIPS enabled SES endpoint (for AWS GovCloud).

https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/using-govcloud-endpoints.html

**Comment by kapilt** [comment]
that would need a code change and exposing via configuration file, contributions welcome.
